src/sds.c

```diff
@@ -560,7 +560,7 @@ sds sdscatfmt(sds s, char const *fmt, ...) {
  * s = sdstrim(s,"Aa. :");
  * printf("%s\n", s);
  *
- * Output will be just "Hello World".
+ * Output will be just "a.aa.aHello World".
```

Comment:
From what I see, in the example we pass both upper and lower case A: 'Aa', so there's no reason for this doc fix.